Miscellaneous/Problems/Solution/Ayush/Ques4.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Scanner;
+public class Ques4 {
+    public static void main(String[] args) {
+        System.out.println("The unit for input");
+        System.out.println("Press 1 for Celsius");
+        System.out.println("Press 2 for Fahrenheit");
+        System.out.println("Press 3 for Kelvin");
+        Scanner input = new Scanner(System.in);
+
+        int choice = input.nextInt();
+
+        System.out.println("Convert to which scale");
+        System.out.println("Press 1 for Celsius");
+        System.out.println("Press 2 for Fahrenheit");
+        System.out.println("Press 3 for Kelvin");
+        int choice2 = input.nextInt();
+        System.out.println("Enter the temperature");
+        float temp = input.nextFloat();
+        float cf = temp*((float)(9/5) + 32);
+        float ck = temp+(float)273.15;
+        float fk = (float)((float)(temp-32)*((float)5/9)+273.15);
+        float fc = (float)(temp-32)*((float) 5 /9);
+        float kc = (float) (temp-273.15);
+        float kf = (float)((float)(temp-273.15)*((float)9/5)+32);
+
+
+        switch (choice){
```

Comment:
check the logic @Ayush8087 

I mean it seems fair for selection but it would be more practical if you had handled the case of same unit and same scale to convert to

Reply:
so sir do i need to make it more optimized or is it fine??

Reply:
It's not about more optimization, it's about the thinking process where why would one convert to same unit if he already knows that unit?